src/helpers/SendEmail.ts

```diff
@@ -0,0 +1,29 @@
+import nodemailer from "nodemailer";
+
+const transporter = nodemailer.createTransport({
+  service: "gmail",
+  auth: {
+    user: process.env.EMAIL_CURRENCI,
+    pass: process.env.EMAIL_PWD_CURRENCI,
+  },
+});
+
+const sendEmail = (to, subject, text, html) => {
+  const mailOptions = {
+    from: process.env.EMAIL_CURRENCI,
+    to,
+    subject,
+    text,
+    html,
+  };
+
+  transporter.sendMail(mailOptions, (error, info) => {
```

Comment:
Add error handling for transporter.sendMail to capture cases where the email server might be down.